src/core/stream/representation/representation_stream.ts

```diff
@@ -301,9 +301,9 @@ export default function RepresentationStream<TSegmentDataType>({
                    IStreamNeedsManifestRefresh |
                    IStreamTerminatingEvent>
     {
-      const wantedStartPosition = observation.position + observation.wantedTimeOffset;
+      const initialWantedTime = observation.position + observation.wantedTimeOffset;
```

Comment:
I found `initialWantedTime` was a clearer name than `wantedStartPosition` (because it automatically leads to the following question: "start of what?"!)

`initialWantedTime` like its predecessor is the initial position where segments should be pushed to the buffer. It is generally equal to the current position, with exceptions, such as when the content did not start yet, In that last situation, it is for example equal to the initially wanted time, not the current position which is generally `0` for HTML5-related reasons.